src/app/container/launch/tool-launch.service.spec.ts

```diff
@@ -41,14 +41,18 @@ describe('LaunchService', () => {
 
     it('should getCWLString', inject([ToolLaunchService], (service: ToolLaunchService) => {
         expect(service.getCwlString('quay.io/a/b', 'c'))
-            .toContain('cwltool --non-strict');
+            .toContain('cwl-runner --non-strict');
```

Comment:
`--non-strict` shouldn't be here. It isn't part of the CWL standard and descriptions that need it are broken. I opened several PRs for prominent tools posted on dockstore.org to fix the "need" for `--non-strict" :-)

Reply:
Thanks for the heads-up!
I think I'll have to take a look for those PRs, I may have missed the notifications. In the meantime, I agree that we cannot suggest `--non-strict` for cwl-runner if it only applies for cwltool. 

@garyluu  just note that `non-strict` or its equivalent for other engines can be a troubleshooting step whether via tooltip or shell comment